### ISSUE TITLE
Add script to create and store MP4 clips corresponding to an API request for events 

### ISSUE THREAD

**Issue by monocongo** [monocongo/video_utils#16]
We need a script that we can use to send an API request for all events detected within a time period and then for each event in the response we create an MP4 clip based on the event timestamp and store the clip to S3 bucket.

An example API endpoint: `http://admiraldemo.scw-ddns.com:80/LAPI/V1.0/Channels/1/Media/Video/Streams/1/Records?Begin=1560052800&End=1560139199`

**Comment by monocongo** [comment]
The script can be used at the command line like so:

```bash
$ python clip_and_store_events.py --device admiraldemo.scw-ddns.com --begin 1560052800 --end 1560139199 --user admin --password opensesame --s3_bucket scw.james.adams --s3_prefix events_test
```

**Comment by monocongo** [comment]
Resolved by PR #17
